tests/apps/assessment/test_assessment_models.py

```diff
@@ -0,0 +1,39 @@
+import pytest
+from django.test.client import Client
+from django.urls import reverse
+from pytest_django.asserts import assertTemplateUsed
+
+from hawc.apps.assessment.models import Blog
+
+
+@pytest.mark.django_db
+def test_assessment_creation():
```

Comment:
`test_assessment_creation` think this one can be deleted; it's been moved from the dtxid branch